### ISSUE TITLE
missing /storage/*/xtrMapper.sh

### ISSUE THREAD

**Issue by izryel** [Xtr126/XtMapper#34]
no such file at /storage/emulated/0/Android/data/xtr.keymapper/files/xtrMapper.sh

not even in the APK when quick decompress nor on the repo here for manual install...

**Comment by Xtr126** [comment]
> no such file at /storage/emulated/0/Android/data/xtr.keymapper/files/xtrMapper.sh
> 
> not even in the APK when quick decompress nor on the repo here for manual install...

It is `xtMapper.sh` not `xtrMapper.sh`
I was assuming that it was fixed in  #6, not sure whats wrong in your system. Cannot reproduce on my end.
Can you provide some details on what type of device you are using the app on (phone/tablet or PC) and what Android ROM. 
The location of application files in /data/app is randomized by Android during installation of an APK. 
The script is generated by the app at runtime with the path to the application files. It is not pre-defined.
But the following command may work instead of the script (run as root using su or otherwise):
```
env LD_LIBRARY_PATH=$(echo /data/app/*/xtr.keymapper*/lib/x86_64) CLASSPATH=$(echo /data/app/*/xtr.keymapper*/base.apk) /system/bin/app_process /system/bin xtr.keymapper.server.InputService
```

**Comment by Xtr126** [comment]
Updated the docs with the right command ( above command is invalid ): https://xtr126.github.io/XtMapper-docs/guides/activation
